src/flexalloc_daemon.c

```diff
@@ -85,6 +85,12 @@ msg_handler(struct fla_daemon *d, int client_fd, struct fla_msg const * const re
     if (FLA_ERR(fla_daemon_fs_init_rsp(d, client_fd, recv, send), "fla_daemon_init_info()"))
       return -1;
     break;
+  case FLA_MSG_CMD_POOL_SET_STRP:
+    FLA_DBG_PRINT("FLA_MSG_CMD_POOL_SET_STRP\n");
```

Comment:
You probably want to remove the dbg

Reply:
This is a good catch. I put this here because I wanted to be consistent with the DBG line in the FLA_MSG_CMD_INIT_INFO case. I see that the pattern is _NOT_ to have the Debug lines included. So I'll remove Bot the one in FLA_MSG_CMD_INIT_INFO and FLA_MSG_CMD_POOL_SET_STRP.